src/components/common/Picker/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Instead of importing PropTypes you could just import the types you are actually going to use like ```import { object, string, array } from prop-types;```